packages/amplify_auth_cognito/ios/Classes/SwiftAuthCognito.swift

```diff
@@ -293,6 +300,22 @@ public class SwiftAuthCognito: NSObject, FlutterPlugin, FlutterStreamHandler {
       }
     }
     
+    private func onGetCurrentUser(flutterResult: @escaping FlutterResult) {
+      do {
+        guard let user = Amplify.Auth.getCurrentUser() else {
+           // TODO: Use the AuthPluginErrorConstants
+           throw AuthError.signedOut(
+           "You are currently signed out.",
+           "Please sign in and reattempt the operation.")
+        }
+        let userData = FlutterAuthUserResult(res: user)
+        flutterResult(userData.toJSON())
+        
+      } catch {
+          self.handleAuthError(error: error as! AuthError, flutterResult: flutterResult,  msg: FlutterAuthErrorMessage.GET_CURRENT_USER.rawValue)
+        }
```

Comment:
```suggestion
      } catch {
          self.handleAuthError(
              error: error as! AuthError,
              flutterResult: flutterResult,
              msg: FlutterAuthErrorMessage.GET_CURRENT_USER.rawValue
          )
      }
```

Reply:
This is definitely more readble, but I think I should put this in a style PR rather than this one (again, for consistency)